addons/mail/models/mail_mail.py

```diff
@@ -20,6 +20,9 @@
 
 _logger = logging.getLogger(__name__)
 
+BASE_64_SIZE_OVERHEAD_RATIO = 8 / 6
```

Comment:
If we do a small tool function, you will be able to add a small docstring to explain all those computation :D